docs/learn/learn-polkadotjs.md

```diff
@@ -0,0 +1,102 @@
+---
+id: learn-polkadotjs
+title: PolkadotJS
+sidebar_label: PolkadotJS
+description: Learn about PolkadotJS
+slug: ../learn-polkadotjs
+---
+
+<!-- This page is a WIP -->
+<!-- The first version of this page takes motivation from Emre's ELI5 on Polkadot-JS -->
+
+Polkadot-JS is a collection of tools that interfaces with the Polkadot blockchain in a granular 
+way. 
+
+## Primary Implementation
+
+Polkadot-JS as a term has multiple moving parts that are worth mentioning.
+
+### [Polkadot-JS UI](polkadot.js.org)
+
+The Polkadot-JS UI is a hosted application that loads in your browser. The UI has a standard DNS hosted
+version, which always has the latest features, and an IPFS version that is less frequently updated but 
+is more decentralized. This is also often referred to as Polkadot-JS Apps, or the Apps UI.
+
+Polkadot-JS Apps has many capabilities that go beyond basic wallet functions such as account creation 
+and sending or receiving transactions. 
+
+### [Polkadot-JS Extension](https://polkadot.js.org/extension/)
+
+The extension is a simple proof-of-concept for managing accounts in a browser extension and allowing 
+the signing of extrinsics using these accounts. The Polkadot-JS extension is not made for users to 
+interact with on-chain functions as one would find through a wallet app. The extension acts as 
+a robust key-store and thus acts as an account manager for Substrate-based accounts.
+
+However, it also provides a simple interface for interacting with extension-compliant dApps. 
+
+#### [Polkadot-JS Phishing List](polkadot.js.org/phishing/)
+
+The phishing list website is a community-driven curation of less-than-honest operators. The Polkadot-JS 
+extension uses this list to warn a user about suspicious URLs and addresses that are part of the list, and 
+automatically blocks the account address. 
+
+### [Polkadot-JS API](github.com/polkadot-js/api)
+
+A JavaScript API allowing for programs to interface with the functionality of Polkadot.
+
+## Polkadot-JS Apps
+
+Polkadot-JS Apps is the native Polkadot/Kusama/Substrate UI for interacting with a node, allowing access to 
+all features available on Substrate chains.
+
```

Comment:
I would add that it also allows you to interface with a local node, which is useful for developers